specs/core/0_beacon-chain.md

```diff
@@ -1661,6 +1661,11 @@ def process_attestation(state: BeaconState, attestation: Attestation) -> None:
         proposer_index=get_beacon_proposer_index(state),
     )
 
+    # Check bitlist lengths
+    committee_size = get_committee_count(state, attestation.data.target.epoch)
+    assert len(attestation.aggregation_bits) == committee_size
+    assert len(attestation.custody_bits) == committee_size
```

Comment:
I think these lines were missing. While there is a test that was purportedly testing inconsistent bit tests (`test_inconsistent_bits` in `test_process_attestation.py`), it actually also sets some of the custody bits to one, so it actually triggered the `if not len(bit_1_indices) == 0:` check that should not occur in phase 1. So I think these checks should be added.

Reply:
(just noticed there is an issue about this #1262, might be better to fix this in a separate PR)

Reply:
I have a PR coming that handles this in pretty much the same way and had some additional tests. Would like to get it merged as a separate PR in `v08x`

Reply:
Yes, sounds sensible. I'll keep the lines here until that's merged because otherwise it'll fail the tests :D

Reply:
addressed in #1317